### PR TITLE
Add location fields

### PR DESCRIPTION
**Pull request by Baggerone**
Also fixed migration file names so `down` files match their `up` file.

### CODE REVIEW

File: application/models/testutils_test.go [hunk 1]
```diff
@@ -213,10 +213,17 @@ func createPotentialProviderFixtures(tx *pop.Connection, nRequests, nProviders i
...
+	cities := []string{"Miami", "Toronto", "Mexico City", "Chiang Mai", "Paris", "Port Moresby"}
 	locations := make(Locations, n)
+
+	/* #nosec */
 	for i := range locations {
+		randInt := rand.Intn(6)
 		locations[i] = Location{
-			Country:     countries[rand.Intn(6)],
+			Country:     countries[randInt],
+			State:       states[randInt],
```
Comment (Schparky) [inline application/models/testutils_test.go:225]: Looks like we'll get locations like "Mexico City, FL, FR" Reminds me of those kids' books with mix-and-match animal parts, haha!

File: application/migrations/20210712231000_add_fields_to_locations.up.fizz [hunk 2]
```diff
@@ -0,0 +1,5 @@
+add_column("locations", "state", "string")
```
Comment (Schparky) [inline application/migrations/20210712231000_add_fields_to_locations.up.fizz:1]: What did you think of making the lat/long not nullable? Better done in a separate PR?

**Review by Schparky — APPROVED** [review]
Could you fill in some of these fields in the grifts?

File: application/migrations/20210712231000_add_fields_to_locations.up.fizz (hunk 2 above)
Comment (Baggerone) [reply application/migrations/20210712231000_add_fields_to_locations.up.fizz:1]: Yeah, separate PR.

File: application/models/testutils_test.go (hunk 1 above)
Comment (Baggerone) [reply application/models/testutils_test.go:225]: I intended to make the city, state and country all correspond with each other.  This does that doesn't it?  (I.e. Miami will only go with Florida and the US)

Comment (Schparky) [reply application/models/testutils_test.go:225]: Oh, yeah, sorry. I just saw `randInt` on them and my brain though they were all separately random.
